wslib-classes/GUI/Extensions/extServer-makeView.sc

```diff
@@ -172,7 +172,7 @@
 		w.front;
 
 		ctlr = SimpleController(this)
-			.put(\serverRunning, {	if(this.serverRunning,running,stopped) })
+			.put(\this.serverRunning, {	if(this.serverRunning,running,stopped) })
```

Comment:
No, of course this wrong. I've corrected that in a subsequent commit which should be part of my pull request.